flex-commerce-api.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "thor", "~> 0.19"
   spec.add_development_dependency "json_matchers", ["~> 0.5", ">= 0.5.1"]
 
-  spec.add_dependency "oj", "~> 3.3.10"
+  spec.add_dependency "oj", "~> 2.12"
```

Comment:
Can we update this to `2.18`, please?

Reply:
just saw your comment as I was merging - sorry.

Reply:
@mzishtiaq  release `v.0.6.33` uses `v.2.12`. I wanted to keep them similar hence why  didn't update to `v.2.18`